src/main/java/byteceps/ui/strings/ManagerStrings.java

```diff
@@ -30,11 +30,20 @@ public class ManagerStrings {
     public static final String INCOMPLETE_EDIT = "Edit command not complete. " +
             "Please use: exercise /edit <OLD_EXERCISE_NAME [string]> /to <NEW_EXERCISE_NAME [string]>";
 
+    public static final String INCOMPLETE_DELETE_EXERCISE = "Delete command not complete. " +
+            "Please use : exercise /delete <EXERCISE_NAME [string]>";
+
     // Workout strings
     public static final String WORKOUT = "Workout";
     public static final String WORKOUTS = "Workouts";
-    public static final String INCOMPLETE_INFO = "info command not complete. " +
+    public static final String INCOMPLETE_INFO = "Info command not complete. " +
             "Please use: workout /info <WORKOUT_PLAN_NAME [string]> ";
+    public static final String INCOMPLETE_CREATE = "Create command not complete. " +
+            "Please use: workout /create <NEW_WORKOUT_PLAN_NAME [string]>";
+    public static final String INCOMPLETE_DELETE_WORKOUT = "Delete command not complete." +
+            "Please use: workout /delete <WORKOUT_PLAN_NAME [string]>";
+    public static final String INCOMPLETE_SEARCH = "Search term cannot be left blank." +
+            "Please use : workout /search <WORKOUT_PLAN_NAME [string]>";
```

Comment:
Might want to standardize if there's a space before the colon or not (also in line 34)